k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-ha/values.yaml

```diff
@@ -0,0 +1,30 @@
+namespace: default
+cortxha:
+  name: cortx-ha
+  image: ghcr.io/seagate/centos:7
+  secretinfo: secret-info.txt
+  serviceaccountname: cortx-sa
```

Comment:
can define different name possible 'cortx-ha'
as for ha pod, we need to give some more permissions through clusterrole and clusterrolebinding.
it is already part of some other ticket @osowski as it is required to watch on pods for getting events.
so kipping a separate account will give this permission to HA pod only.

Reply:
Can you create a JIRA ticket for the specific request of creating a new Service Account specific to the HA pod and the explicit permissions it requires to run with (above and beyond the rest of the basic permissions the existing CORTX SA is given)? You can assign it to me directly.

Reply:
it's ok, you will get the required input from the provisioner test, team you can go by your process. :-)